crypto/hash.go

```diff
@@ -1,3 +1,6 @@
+// Package crypto is for an ethereum hash (Keccak256) wrapper that returns a 32-byte array
```

Comment:
Package crypto is for cryptographic code used by both Monero and Ethereum. Chain specific crypto is in subpackages.